engines/qemu/network/pool.go

```diff
@@ -362,8 +368,9 @@ func destroyNetwork(n *entry) error {
 		{"ip", "addr", "del", n.ipPrefix + ".1", "dev", n.tapDevice},
 		// Delete tap device
 		{"ip", "tuntap", "del", "dev", n.tapDevice, "mode", "tap"},
-	})
+	}, true)
 	if err != nil {
+		debug("Failed to destory tap device: %s, error: %s", n.tapDevice, err)
```

Comment:
This code is so scary, I think it warrants a few more comments to explain the finer points, or give an overview of how the network gets configured, and why. For most people coming fresh to this, it would take a lot of study/investigation to understand it.